src/frontends/onnx/frontend/src/version_range.hpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+namespace ngraph {
+namespace onnx_import {
+static constexpr const int LATEST_SUPPORTED_ONNX_OPSET_VERSION = ONNX_OPSET_VERSION;
+struct VersionRange {
+    VersionRange(int from, int to) : m_from(from), m_to(to) {}
+    static VersionRange from_version(int from) {
+        return VersionRange{from, LATEST_SUPPORTED_ONNX_OPSET_VERSION};
+    }
+    static VersionRange to_version(int to) {
```

Comment:
```suggestion
    static VersionRange until_version(int version) {
```
Here on the other hand we would be aligned with how cppreference defines things https://en.cppreference.com/w/cpp/container/vector/vector

Reply:
Or make it even shorter name `since`, `until` and call like `VersionRange::until` instead `VersionRange::until_version` (used version word twice).

Reply:
@tomdol @praasz I like it, done, thanks